en-US/mozorg/internet-health/index.lang

```diff
@@ -103,19 +104,40 @@ The Internet was built on the promise that any one of us might create the next b
 Support Internet Openness
 
 
+## TAG: health_hub_common_voice
+;Common Voice
+Common Voice
+
+
+## TAG: health_hub_common_voice
+# Link points to https://voice.mozilla.org/
+;Let your voice be heard… literally
+Let your voice be heard… literally
+
+
+## TAG: health_hub_common_voice
+;Contribute to the world’s first crowdsourced voice library. An open, free resource, it will help innovators create technology that benefits — <em>and understands</em> — us all.
+Contribute to the world’s first crowdsourced voice library. An open, free resource, it will help innovators create technology that benefits — <em>and understands</em> — us all.
+
+
+## TAG: health_hub_common_voice
+# Link points to https://voice.mozilla.org/
+;Record in Common Voice
+Record in Common Voice
+
+
+# String is obsolete. Do not remove.
 # This is a link
 ;Net Neutrality
 Net Neutrality
 
 
+# String is obsolete. Do not remove.
 ;Net Neutrality is under threat…again.
 Net Neutrality is under threat…again.
 
 
-;Time’s running out. Help protect an open Internet for everyone.
```

Comment:
Uhm, what happened here? The request was to add the obsolete comment to the next string, but it looks like you removed it?